scripts/module/keybinder.nut

```diff
@@ -1,36 +1,41 @@
-keys <- {
-  "a" : 0x41,
-    "b" : 0x42,
-    "c" : 0x43,
-    "d" : 0x44,
-    "e" : 0x45,
-    "f" : 0x46,
-    "g" : 0x47,
-    "h" : 0x48,
-    "i" : 0x49,
-    "j" : 0x4A,
-    "k" : 0x4B,
-    "l" : 0x4C
-    }; // FIXME: Complete the table
-
-
+dofile('./keytable.nut');
 binds <- array(100, null);
 
+function extractFirstWord( input ) {
```

Comment:
Why not make this more powerful and standard? `partition` with a `separator` parameter?

Also see: https://docs.python.org/3/library/stdtypes.html#str.partition for example.

(although I'd only return 2 values - I always found that returning 3 values in Python is a design flaw)